tox.ini

```
@@ -4,9 +4,7 @@ isolated_build = True
 
 [testenv]
 description = Run pytest against {envname}.
-deps =
-    -r requirements/main.txt
-    -r requirements/dev.txt
+allowlist_externals = pytest
```

```suggestion
extras =
    dev
```

This means that these tox environments will install the `dev` extra dependencies in pyproject.toml in addition to the core dependencies. Since `pytest` should be in those `dev` dependencies, you don't need/want to use an external pytest command.